ifrs17/Report/ReportConfigurationAndUtils.ipynb

```diff
@@ -50,11 +50,12 @@
                 "\n    int headerColumnWidth = 250,",
                 "\n    int groupDefaultExpanded = 2)",
                 "\n    => pivotBuilder.ToTable().WithOptions(go => go",
-                "\n                                     .WithColumns(cols => cols.Modify(\"Value\",c => c.WithWidth(valueColumnWidth).WithFormat(\"new Intl.NumberFormat('en',{ minimumFractionDigits:2, maximumFractionDigits:2 }).format(value)\")))",
+                "\n                                     .WithColumns(cols => cols.Modify(\"Value\",c => c.WithWidth(valueColumnWidth).WithFormat(\"typeof(value) == 'number' ? new Intl.NumberFormat('en',{ minimumFractionDigits:2, maximumFractionDigits:2 }).format(value) : value\")))",
                 "\n                                     .WithRows(rows => rows",
                 "\n                                               .Where(r => !(r.RowGroup.Coordinates.Last() == \"NullGroup\"))",
                 "\n                                               .Select(r => r with { RowGroup = r.RowGroup with { Coordinates = r.RowGroup.Coordinates.Where(c => c != \"NullGroup\").ToImmutableList() } })",
                 "\n                                               .ToArray())",
+                "\n                                      .HideRowValuesForDimension(\"Novelty\")",
```

Comment:
I see this....
If you collapse the group, would you still see the info.
Pity to let it go completely, but should improve the understanding.

Reply:
Collapsing the novelty does not show and aggregation.